README.md

```diff
@@ -29,6 +29,20 @@ The Microsoft Graph JavaScript client library is a lightweight wrapper around th
         -   [Page Iteration](docs/tasks/PageIterator.md)
         -   [Getting Raw Response](docs/GettingRawResponse.md)
         -   [Creating an instance of TokenCredentialAuthenticationProvider](docs/TokenCredentialAuthenticationProvider.md)
+    - [Samples and tutorials](#samples-and-tutorials)
+        - Step-by-step training exercises on creating a basic application using the Microsoft Graph JavaScript SDK:
+            - [Build Angular single-page apps with Microsoft Graph](https://docs.microsoft.com/en-us/graph/tutorials/angular)
```

Comment:
nit: if you remove en-us from the links, people will get redirected to their language if it's available